### ISSUE TITLE
Unable to Export PDF in VSCode 1.63.0

### ISSUE THREAD

**Issue by RemyPorter** [piersdeseilligny/betterfountain#147]
HTML export works fine, but when I attempt to export to PDF the title page exports successfully, and if the first thing after the title page is a transition, that exports correctly, but nothing else. The PDF just has a title page, a transition, and nothing after the transition.

VSCode version info:
```Version: 1.63.0
Commit: 7db1a2b88f7557e0a43fec75b6ba7e50b3e9f77e
Date: 2021-12-07T05:16:45.441Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.4.0-90-generic snap
```

I'm running on Elementary OS 5, with uname info: `Linux bismuth 5.4.0-90-generic #101~18.04.1-Ubuntu SMP Fri Oct 22 09:25:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux`

Sample failing script:

```
Title:
    _**Rowboat Cop**_
Credit: Written by
Author: Remy Porter


### Scene 1

EXT. A city street. NIGHT.

> LIGHTS UP.

A pair of grungy, 80s movie thugs threaten a woman in a pants-suit. It's very much a leering, over-the-top presentation of an 80s movie thug, they've got butterfly knives and chains and walked out of the movie Robocop.

THUG1
Why don't you hand over what's in the purse and we don't have to hurt you.

THUG2
*Much*

VICTIM
(screams)
Help! Police! Help!
(to the thugs)
Roboat Cop will save me!

ROWBOAT COP
(offstage, robotically)
Dead or alive, you're coming with me.

The thugs react, turning towards the voice. At first, they're frightened and look like they're about to bolt. But after a beat, they just look puzzled.
Rowboat Cop enters by rowing his way in. He's in a rowboat. On land. Tugging at oars. He's just a cop in a rowboat.

> BLACKOUT
```

The `exthost.log` has this error trace:

```
[2021-12-09 16:57:10.770] [exthost] [error] [piersdeseilligny.betterfountain] provider FAILED
[2021-12-09 16:57:10.771] [exthost] [error] Error: selectionRange must be contained in fullRange
	at Function.validate (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:86:30642)
	at new t (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:86:30510)
	at new t (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:86:17301)
	at symbolFromStruct (/home/remy/.vscode/extensions/piersdeseilligny.betterfountain-1.8.10/out/providers/Symbols.js:31:26)
	at symbolFromStruct (/home/remy/.vscode/extensions/piersdeseilligny.betterfountain-1.8.10/out/providers/Symbols.js:36:39)
	at symbolFromStruct (/home/remy/.vscode/extensions/piersdeseilligny.betterfountain-1.8.10/out/providers/Symbols.js:36:39)
	at FountainSymbolProvider.provideDocumentSymbols (/home/remy/.vscode/extensions/piersdeseilligny.betterfountain-1.8.10/out/providers/Symbols.js:49:26)
	at s.provideDocumentSymbols (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:95:87725)
	at /snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:95:112809
	at We._withAdapter (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:95:111974)
	at We.$provideDocumentSymbols (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:95:112785)
	at t._doInvokeHandler (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:99:13802)
	at t._invokeHandler (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:99:13486)
	at t._receiveRequest (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:99:12097)
	at t._receiveOneMessage (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:99:10826)
	at /snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:99:8922
	at u.fire (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:66:1712)
	at r.fire (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:74:18942)
	at /snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:114:34341
	at u.fire (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:66:1712)
	at r.fire (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:74:18942)
	at n._receiveMessage (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:74:23523)
	at /snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:74:21057
	at u.fire (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:66:1712)
	at w.acceptChunk (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:74:15770)
	at /snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:74:14900
	at Socket.T (/snap/code/83/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:114:13813)
	at Socket.emit (events.js:315:20)
	at Socket.EventEmitter.emit (domain.js:467:12)
	at addChunk (internal/streams/readable.js:309:12)
	at readableAddChunk (internal/streams/readable.js:284:9)
	at Socket.Readable.push (internal/streams/readable.js:223:10)
	at Pipe.onStreamRead (internal/stream_base_commons.js:188:23)
```

I suspect this has something to do with me running VSCode as a snap, but I'm not certain.

**Comment by piersdeseilligny** [comment]
Hi,
Thanks for reporting this - I'm running into the same issue with that sample script, so it's definitely an issue with betterfountain.

As far as I can tell, it's related to the screenplay starting with a subsection (`### Scene 1`). As a temporary workaround, the PDF exports correctly when starting it with a regular section (`# Scene 1`), or when disabling the "Create bookmarks for each section/scene" setting.

I'il publish a fix asap.

**Comment by piersdeseilligny** [comment]
Fixed in 1.8.11, which is now live in the marketplace :)

**Comment by RemyPorter** [comment]
Fantastic! It's a great tool and super useful for me. I appreciate the work!
